src/components/Dropdown/V2/Dropdown.Item.tsx

```diff
@@ -0,0 +1,225 @@
+import * as React from 'react'
+import getValidProps from '@helpscout/react-utils/dist/getValidProps'
+import propConnect from '../../PropProvider/propConnect'
+import Flexy from '../../Flexy'
+import Icon from '../../Icon'
+import Menu from './Dropdown.Menu'
+import {
+  ItemUI,
+  ActionUI,
+  WrapperUI,
+  SubMenuIncidatorUI,
+} from './Dropdown.css.js'
+import {
+  SELECTORS,
+  setMenuPositionStyles,
+  getCustomItemProps,
+} from './Dropdown.utils'
+import { classNames } from '../../../utilities/classNames'
+import {
+  getComponentKey,
+  namespaceComponent,
+} from '../../../utilities/component'
+import { noop } from '../../../utilities/other'
+import { COMPONENT_KEY } from './Dropdown.utils'
+
+export interface Props {
+  actionId?: string
+  className?: string
+  disabled: boolean
+  dropRight: boolean
+  dropUp: boolean
+  id?: string
+  index: string
+  innerRef: (node: HTMLElement) => void
+  isHover: boolean
+  items: Array<any>
+  onMouseEnter: (event: Event) => void
+  onMouseMove: (event: Event) => void
+  onBlur: (event: Event) => void
+  onClick: (event: Event, props: any) => void
+  onFocus: (event: Event) => void
+  renderItem?: (props: any) => void
```

Comment:
I know having strong types is good, but I think noop functions are one case where typed arguments are not important, what about using something like `(...args: any[]) => void` and creating a `noop` util to clean this up a bit, since now writing one each time is a bit more verbose?

Reply:
@knicklabs TOTALLY did not know you can do that. Just gave it a shot, and it worked! (At least in VS Code) 😍